### PR TITLE
Update fn_spawnVehicleGroup.sqf

### PR DESCRIPTION
**Pull request by Joecuronium**
added automatic teleporting of units into zeus-ui-spawned vehicles, should facilitate quicker vehicle ingress times with less micromanaging

### CODE REVIEW

**Comment by Bubbus** [comment]
Looks good, thanks.

**Comment by dgibso29** [comment]
wow i did not approve this what kind change process is this wow

File: components/spawnNpcs/fn_spawnVehicleGroup.sqf [hunk 1]
```diff
@@ -37,6 +37,7 @@ _gunno = true;
         if (!_check) then
         {
             _x assignAsCommander _vehicle;
+            _x moveInCommander _vehicle;
```
Comment (dgibso29) [inline components/spawnNpcs/fn_spawnVehicleGroup.sqf:40]: Very nice

**Comment by dgibso29** [comment]
If I could click an "Approved" button, I would.

**Pull request merged**